backend/src/modules/boards/services/get.board.service.ts

```diff
@@ -290,31 +291,29 @@ export default class GetBoardServiceImpl implements GetBoardServiceInterface {
 	> {
 		const { hideCards = false, hideVotes = false, columns: boardColumns } = input;
 
-		if (hideCards || hideVotes) {
-			// Columns
-			const columns = boardColumns.map((column) => {
-				const cards = column.cards.map((card) => {
-					const items = card.items.map((item) => {
-						return this.replaceCard(item, userId, hideCards, hideVotes);
-					});
-
-					return {
-						...this.replaceCard(card, userId, hideCards, hideVotes),
-						items
-					};
+		// Columns
+		const columns = boardColumns.map((column) => {
+			const cards = column.cards.map((card) => {
+				const items = card.items.map((item) => {
+					return this.replaceCard(item, userId, hideCards, hideVotes);
 				});
 
 				return {
-					...column,
-					cards
+					...this.replaceCard(card, userId, hideCards, hideVotes),
+					items
 				};
 			});
 
 			return {
-				...input,
-				columns
+				...column,
+				cards
 			};
-		}
+		});
+
+		return {
+			...input,
+			columns
+		};
 
 		return input;
```

Comment:
```suggestion
		return {
			...input,
			columns
		};
```

You can remove the last return, because already have one.